docs/src/tutorials/software-management.md

```diff
@@ -70,23 +40,23 @@ More info about managing the device software can be found [here](https://cumuloc
    - to add a new software, enter a name for the software (and confirm it by clicking Create new in the resulting window),
      a description and its version.
    - to add a new version, select the software for which you want to add a new version from the dropdown list in the Software
-     field and enter a version and plugin type as **version::apt** (In the picture below the version is empty and are optional)
+     field and enter a version of the package. (The version is optional and can be left with a white space)
 3. Optionally, you can define the device type filter when adding a new software.
-4. As of now thin-edge.io only supports software from apt repositories. So, select the **Provide a file path file path** option and give an **empty space**.
+4. As of now thin-edge.io only supports software from apt repositories. So, select the **Provide a file path** option and give an **empty space**.
```

Comment:
I would add a note that the plugin type name that is provided here is used to pick the appropriate plugin among those installed in `/etc/tedge/sm-plugins`.